README.md

```diff
@@ -56,44 +53,30 @@ A simplified manifest would look like this:
   "name": "My Company",
   "implementation": "My implementation",
   "issuers": [{
-    "id": "",
-    "endpoint": "https://mycompany.example/credentials/issue",
-    "method": "POST",
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"]
-    "tags": ["ecdsa-rdfc-2019"]
-  }, {
-    "id": "",
-    "endpoint": "https://mycompany.example/credentials/issue",
-    "method": "POST",
-    "supportedEcdsaKeyTypes": ["P-256"]
-    "tags": ["ecdsa-jcs-2019"]
-  }, {
     "id": "",
     "endpoint": "https://mycompany.example/credentials/issue",
     "method": "POST",
     "supportedEcdsaKeyTypes": ["P-256"]
-    "tags": ["ecdsa-sd-2023"]
+    "tags": ["bbs-2023"]
```

Comment:
Once this is merged we need to be sure to add it to the implementations list of tags: https://github.com/w3c-ccg/vc-api-test-suite-implementations/blob/main/README.md#tags